PoobChat/PoobChat/HomeView.swift

```diff
@@ -9,9 +9,72 @@ import SwiftUI
 
 struct HomeView: View {
     var body: some View {
-        Text("Home View")
+        NavigationView{
```

Comment:
[Style] SwiftUI encourages you to break pieces up into logical view components, we should do that here.

For example:

```swift

var navBar: some View {
    VStack(spacing: 0){
             HStack(){
                     Button {

                     } label: {
                         Image(systemName: "person.fill")
                             .foregroundColor(.black)
                             .font(.system(size: 34))
                     }

                     VStack(){
                         Text("Username")
                             .font(.system(size: 24, weight: .bold))
                     }
                     Spacer()
                     Button {

                     } label: {
                         Image(systemName: "bubble.left")
                             .foregroundColor(.black)
                     }
                 }
                 .padding()
                 Divider()
    }
}
```

The, you can reference these components in the main view, makes it much easier to understand:

```swift
var body: some View {
    VStack(spacing: 0) {
        navBar
        messageList // not shown in my example, you would have to make this
    }
}
```